control-station/src/components/SensorBoxes/Camera&Console/Camera.tsx

```diff
@@ -0,0 +1,8 @@
+import "./style.css"
+export default function Camera(){
+    return(
+        <div id="camera">
+
+        </div>
+    )
+}
```

Comment:
Thought [repeating]: unsure if the Camera and Console would categorically fit under "SensorBoxes". Consider extracting to separate components.